_posts/2016-10-01-cmd-theme.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Cmd Theme
+date: 2016-10-01 23:59:59
+homepage: https://github.com/previoususername/cmd-theme
+download: https://github.com/previoususername/cmd-theme/archive/master.zip
+demo: http://username.tk/
+author: previoususername
+thumbnail: cmd.png
+license: Not determined yet.
```

Comment:
Hello and thanks for the PR! Would You please be so kind and add correct license for the theme.